[2024-11-24T15:18:04.154+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-24T15:18:04.200+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-24T15:17:00+00:00 [queued]>
[2024-11-24T15:18:04.220+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-24T15:17:00+00:00 [queued]>
[2024-11-24T15:18:04.221+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-24T15:18:04.262+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_dependencies> on 2024-11-24 15:17:00+00:00
[2024-11-24T15:18:04.279+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=853) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-24T15:18:04.282+0000] {standard_task_runner.py:72} INFO - Started process 863 to run task
[2024-11-24T15:18:04.284+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection', 'load_dependencies', 'scheduled__2024-11-24T15:17:00+00:00', '--job-id', '589', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_dag.py', '--cfg-path', '/tmp/tmplk9uz51g']
[2024-11-24T15:18:04.288+0000] {standard_task_runner.py:105} INFO - Job 589: Subtask load_dependencies
[2024-11-24T15:18:04.409+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection.load_dependencies scheduled__2024-11-24T15:17:00+00:00 [running]> on host 345e92ff26bf
[2024-11-24T15:18:04.619+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fraud_team' AIRFLOW_CTX_DAG_ID='fraud_detection' AIRFLOW_CTX_TASK_ID='load_dependencies' AIRFLOW_CTX_EXECUTION_DATE='2024-11-24T15:17:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-24T15:17:00+00:00'
[2024-11-24T15:18:04.621+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-24T15:18:04.643+0000] {fraud_detection_dag.py:27} INFO - Chargement des dépendances depuis S3...
[2024-11-24T15:18:38.401+0000] {fraud_detection_dag.py:43} INFO - Modèle chargé avec succès
[2024-11-24T15:18:39.557+0000] {fraud_detection_dag.py:48} INFO - ETL chargé avec succès
[2024-11-24T15:18:40.082+0000] {python.py:240} INFO - Done. Returned value was: True
[2024-11-24T15:18:40.135+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-24T15:18:40.136+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fraud_detection, task_id=load_dependencies, run_id=scheduled__2024-11-24T15:17:00+00:00, execution_date=20241124T151700, start_date=20241124T151804, end_date=20241124T151840
[2024-11-24T15:18:40.206+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-24T15:18:40.791+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-24T15:18:40.801+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
